### ISSUE TITLE
Citra not recognizing landscape mode with Samsung DeX

### ISSUE THREAD

**Issue by Efadd** [citra-emu/citra#7436]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Canary 2779

### Description of Issue

When opening Citra in Samsung DeX, it appears to read as landscape for a moment, but if you change the screens it snaps to portrait

### Expected Behavior

The game screens to be in the landscape orientations

https://github.com/citra-emu/citra/assets/84989803/5f827b97-1f00-427b-8e6c-fc249caa03e2

More videos in #citra-support-android on Discord

### Reproduction Steps

Have a Samsung device with DeX support
Open DeX on an external display
Open Citra and load up a game
Watch as the orientation data is thrown out the window when you try to change the screen setup

### Log File

To be uploaded, mobile sucks

### System Configuration

CPU: SD 888 5G
GPU/Driver: Adreno 660, default drivers
RAM: 8GB
OS: Android 14

**Comment by Efadd** [comment]
![citra_log.txt.old.txt](https://github.com/citra-emu/citra/files/14274377/citra_log.txt.old.txt)

**Comment by ShinyMooTank** [comment]
I also have this issue with my Samsung Dex. The layouts aren't working and if you click swap screen it'll update the screen layout to portrait mode even though default is selected.

Phone: S24 Ultra
CPU: SD Gen 3
GPU/Driver: Adreno 750 , Latest Turnip Drivers. 
RAM: 12GB
OS: Android 14

**Comment by ShinyMooTank** [comment]
Here an image of default being selected but citra is showing the screens as portrait layout. 
![rn_image_picker_lib_temp_5fd4422c-3601-4954-
850a-f5c2439441d3](https://github.com/citra-emu/citra/assets/89095107/c27a6f4e-2c95-425c-9d69-00e23f248584)

**Comment by Efadd** [comment]
Details on Discord, but same issue with SmartView but not Dex using native display

**Comment by Bent3ndo** [comment]
So it’s sounds pretty similar in this case to what is occurring on my Anbernic RG405v on GammaOS. 

If you have single screen as your default mode, a game will start with this being the case. However if you then need to switch to the bottom screen (say to file select) it shifts to dual screen mode (one ontop of the other) and won’t leave that layout no matter how much you cycle. You can screen swap and flip which screen is on top, but you can’t return to a single screen mode.

I’m only using the internal display, not attempting to extend whatsoever. And I’m using L3 and R3 as my inputs for cycling and swapping.

**Comment by Efadd** [comment]
![Screenshot_20240224_081743_Bluecord.jpg](https://github.com/citra-emu/citra/assets/84989803/cc48f837-abd4-444c-bd67-99a9c218575d)

**Comment by ShinyMooTank** [comment]
So basically Citra's Android version doesn't like Android OS that use Landscape as their default view experience. Like Bent3ndo's Anbernic RG405v which runs GammaOS which runs Android OS as landscape. Just like my Samsung Dex/Android desktop mode which the OS is landscape. Citra doesn't check if the android OS is landscaped or not.

**Comment by Bent3ndo** [comment]
> So basically Citra's Android version doesn't like Android OS that use Landscape as their default viewing experience. Like Bent3ndo's Anbernic RG405v which runs GammaOS which runs Android OS as landscape. Just like my Samsung Dex/Android desktop mode which the OS is landscape. Citra doesn't check if the android OS is landscaped or not. 

Yeah basically GammaOS boots as if it's a smartphone on landscape rotation (the GammaOS logo intentionally rotates in 90 degree pulses as a visual illusion to mask it), and then autorotates to the correct orientation before the chosen home app (whether it's ES-DE, Daijishō, etc) visibly boots.

So if Citra could auto correct for  any orientation and then allow cycling of screens still, that would be brilliant.

**Comment by Bent3ndo** [comment]
So when forcing Citra into the orientation the screen is originally designed for (before GammaOS auto rotates), cycling works again.

![image](https://github.com/citra-emu/citra/assets/87569776/eac7cb4a-1fe1-47d9-8d8c-2f29db3b4ed7)

![image](https://github.com/citra-emu/citra/assets/87569776/6a0108a3-8a0c-4cd4-ac48-3c3f99df582d)

![image](https://github.com/citra-emu/citra/assets/87569776/6509743e-8b5a-4f6c-ae6d-a8bea8e220e4)
